src/main/java/backend/UpdateController.java

```diff
@@ -87,7 +89,13 @@ private void openRepositoriesInFilters(List<FilterExpression> filterExprs) {
         return filterExprs.stream()
                 .filter(Qualifier::hasUpdatedQualifier)
                 .flatMap(filterExpr -> allModelIssues.stream()
-                        .filter(issue -> Qualifier.process(models, filterExpr, issue)))
+                        .filter(issue -> {
+                            try {
+                                return Qualifier.process(models, filterExpr, issue);
+                            } catch (SemanticException e) {
+                                return false;
+                            }
+                        }))
```

Comment:
@dariusf Okay, that approach seems to be better. Btw, this warning also shows all errors related to `SemanticException`. Is this intended/OK?

<img width="276" alt="screen shot 2016-03-08 at 19 08 16" src="https://cloud.githubusercontent.com/assets/9299240/13600004/24727764-e561-11e5-8e21-d89c32002cec.png">